Duplicate bridge methods

I get duplicate bridge methods generated by the bridge-method-injector when I compile with JDK 8 resulting in errors like these 

```
[ERROR] java.lang.ClassFormatError: Duplicate method name&signature in class file com/mysema/query/sql/ProjectableSQLQuery
```

It appears JDK 8 copies annotations to bridge methods as well https://bugs.openjdk.java.net/browse/JDK-6695379 

So in cases where the compiler generates a bridge methods for an annotated method there will be two bridge methods generated by this plugin, one delegating to the original method and one to the bridge method.

Would it be possible to add a duplicate filter or skip bridge method target methods?

--- a/injector/src/main/java/com/infradna/tool/bridge_method_injector/MethodInjector.java
+++ b/injector/src/main/java/com/infradna/tool/bridge_method_injector/MethodInjector.java
@@ -250,7 +250,7 @@ public class MethodInjector {
                 @Override
                 public AnnotationVisitor visitAnnotation(String adesc, boolean visible) {
                     AnnotationVisitor av = super.visitAnnotation(adesc, visible);
-                    if (adesc.equals(WITH_SYNTHETIC_METHODS))
+                    if (adesc.equals(WITH_SYNTHETIC_METHODS) && (access & ACC_SYNTHETIC) == 0)
                         return new WithBridgeMethodsAnnotationVisitor(av) {
                         
                             @Override

